app/webhook.go

```diff
@@ -165,10 +165,12 @@ func (a *App) doOutgoingWebhookRequest(url string, body io.Reader, contentType,
 	req.Header.Set("Content-Type", contentType)
 	req.Header.Set("Accept", "application/json")
 
-	now := strconv.FormatInt(time.Now().Unix(), 10)
-	hmacDigest := model.GenerateHmacSignature(&bodyBytes, now, secretToken)
-	req.Header.Set("Timestamp", now)
-	req.Header.Set("Signature", hmacDigest)
+	if len(secretToken) != 0 { //skip signing the outgoing hooks with no secretToken
```

Comment:
nit: while informal, we don't usually inline comments like this. Suggest
```suggestion
        // Skip signing legacy outgoing webhooks with no defined secret token
	if len(secretToken) != 0 { 
```